### DISCUSSION TITLE
Read and Plot 3D Mesh grid and Results from NX Nastran

### DISCUSSION THREAD

**Discussion by drakegeo** [pyvista/pyvista#3084] [Q&A]
Dear all,
Is there anyone who has read and plot results from the commercial software NX Nastran? 
I would like to reconstruct the mesh grid (.dat file) and map the results on the nodes.
Thank you in advance.
George

**Comment by banesullivan** [comment]
Hm. I've never heard of this software. Could you perhaps provide documentation on their file format and provide example data? Working with the outputs of commercial software is always a fun time -- as it will require a bit of investigative work to decipher what's really going on in the file format.
